src/test/munit/sftp-directory-listener-reconnection-test-case.xml

```diff
@@ -27,7 +27,8 @@
     <!-- TODO: SFTPC-9
         <munit:test name="listenerProcessesAllFiles" ignore="#[true]" >
     -->
-    <munit:test name="listenerProcessesAllFiles" >
+    <munit:test name="
+    listenerProcessesAllFiles" >
```

Comment:
This seems unintentional. Roll it back please

Reply:
done.